diff --git a/src/ToasterBase.tsx b/src/ToasterBase.tsx
index 2dcb15cab651cea3de0d898214e84773c89e4ad3..20b0e27a187bde06cf3d99cc80e61bf884f8eef5 100644
--- a/src/ToasterBase.tsx
+++ b/src/ToasterBase.tsx
@@ -96,7 +96,7 @@ const ToasterBaseWithoutRef = <T extends object>(
       pointerEvents="box-none"
     >
       <GestureDetector gesture={panGesture}>
-        <Animated.View onLayout={onLayout}>
+        <Animated.View pointerEvents="box-none" onLayout={onLayout}>
           {[...toasts].reverse().map((e, index) => {
             const hide = () => _hide(e.id);
             return (
diff --git a/src/components/Swipeable/Swipeable.tsx b/src/components/Swipeable/Swipeable.tsx
index 91b45a95ca8fff1e835377f5bcd9b800575fc642..c288eb4e33136cdc179cf0b5099b2158982b46d5 100644
--- a/src/components/Swipeable/Swipeable.tsx
+++ b/src/components/Swipeable/Swipeable.tsx
@@ -50,7 +50,7 @@ export const Swipeable = memo(
 
     return (
       <GestureDetector gesture={panGesture}>
-        <Animated.View style={swipeableStyle}>{children}</Animated.View>
+        <Animated.View pointerEvents="box-none" style={swipeableStyle}>{children}</Animated.View>
       </GestureDetector>
     );
   }
diff --git a/src/components/ToastContainer/ToastContainer.tsx b/src/components/ToastContainer/ToastContainer.tsx
index 4ccb85082d747fa689c852ad0ade139298f05c13..19c62adb9c10fad748bc90dc662a1a4758b2fe2f 100644
--- a/src/components/ToastContainer/ToastContainer.tsx
+++ b/src/components/ToastContainer/ToastContainer.tsx
@@ -46,6 +46,7 @@ export const ToastContainer = memo(
 
     return (
       <Animated.View
+        pointerEvents="box-none"
         style={[styles]}
         entering={entering}
         exiting={exiting}
